app/controllers/messages_controller.rb

```diff
@@ -146,12 +146,12 @@ def quote
     content = "#{ll(Setting.default_language, :text_user_wrote, user)}\\n> "
     content << text.to_s.strip.gsub(%r{<pre>((.|\s)*?)</pre>}m, '[...]').gsub('"', '\"').gsub(/(\r?\n|\r\n?)/, '\\n> ') + '\\n\\n'
     render(:update) { |page|
-      page << "$('message_subject').value = \"#{subject}\";"
-      page.<< "$('message_content').value = \"#{content}\";"
+      page << "$('reply_subject').value = \"#{subject}\";"
+      page.<< "$('reply_content').value = \"#{content}\";"
       page.show 'reply'
```

Comment:
That's actually because of another issue I tried to fix. But I reverted the change for now.

The problem is, that the forum message text area has the ID `message_content` when you create a topic and `reply_content` when you reply to a topic or post. The [wiki toolbar](https://github.com/opf/openproject/blob/dev/app/views/messages/_form.html.erb#L49) is only added for the ID `message_content`. Which means you will have it when you create a topic, but not when you answer to one.

I'm thinking instead of changing the controller code to `reply_*` I should probably change the markup back to `message_*` even for replies. Although I'm not sure if there was a more important reason to rename it to begin with.